build/launchSettings.json

```diff
@@ -15,8 +15,8 @@
       "nativeDebugging": true,
       "environmentVariables": {
         "IIS_SITE_PATH": "$(MSBuildThisFileDirectory)",
-        "ANCM_PATH": "$(AncmPath)",
-        "ANCMV2_PATH": "$(AncmV2Path)",
+        "ANCM_PATH": "$(TargetDir)\\$(AncmPath)",
```

Comment:
This change is to use applocal aspnetcorev2.dll and outofproc handler next to it.